ruby-challenges.rb

```diff
@@ -3,32 +3,92 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
+## Create a function to determine whether a number is even or odd
+# Input: a number (float or integer)
+# Output: a string concatenating input with a statement of even/oddness
+# Function returns a catchall phrase if the input was not an integer
+
+def odd_ball(num)
+      if num % 2 == 0
```

Comment:
Ruby has a handy shorthand method for checking if a number is even or odd.  If you are curious, look up even? and .odd? methods

Reply:
Yes, I saw those during development of our text-based game. Somewhat simpler solution. Half of Ruby is looking up if there's a built-in method for something. The other half seems to be figuring out how I made a syntactical error.